website/source/guides/operations/deployment-guide.html.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: "guides"
+page_title: "Vault Deployment Guide - Guides"
+sidebar_current: "guides-operations-deployment-guide"
+description: |-
+  This deployment guide covers the steps required to install and
+  configure a single HashiCorp Vault cluster as defined in the
+  Vault Reference Architecture
+---
+
+# Vault Deployment Guide
+
+This deployment guide covers the steps required to install and configure a single HashiCorp Vault cluster as defined in the [Vault Reference Architecture](https://www.vaultproject.io/guides/operations/reference-architecture.html).
+
+Below are instructions for installing and configuring Vault on Linux hosts running the systemd system and service manager.
+
+## Reference Material
+
+This deployment guide is designed to work in combination with the [Vault Reference Architecture](https://www.vaultproject.io/guides/operations/reference-architecture.html). Although not a strict requirement to follow the Vault Reference Architecture, please ensure you are familiar with the overall architecture design; for example installing Vault on multiple physical or virtual (with correct anti-affinity) hosts for high-availability and using Consul for the HA and storage backend.
+
+During the installation of Vault you should also review and apply the recommendations provided in the [Vault Production Hardening](https://www.vaultproject.io/guides/operations/production.html) guide.
+
+## Overview
+
+To provide a highly-available single cluster architecture, we recommend Vault be deployed to more than one host, as shown in the [Vault Reference Architecture](https://www.vaultproject.io/guides/operations/reference-architecture.html), and connected to a Consul cluster for persistent data storage.
+
+![Reference Diagram](https://www.vaultproject.io/assets/images/vault-ref-arch-2-02305ae7.png "Reference Diagram")
+
+The below setup steps should be completed on all Vault hosts.
+
+- [Download Vault](#download-vault)
+- [Install Vault](#install-vault)
+- [Configure systemd](#configure-systemd)
+- [Configure Consul](#configure-consul)
+- [Configure Vault](#configure-vault)
+- [Start Vault](#start-vault)
+
+## Download Vault
+
+Precompiled Vault binaries are available for download at [https://releases.hashicorp.com/vault/](https://releases.hashicorp.com/vault/) and Vault Enterprise binaries are available for download by following the instructions made available to HashiCorp Vault customers.
+
+You should perform checksum verification of the zip packages using the SHA256SUMS and SHA256SUMS.sig files available for the specific release version. HashiCorp provides [a guide on checksum verification](https://www.hashicorp.com/security.html) for precompiled binaries.
+
+```text
+VAULT_VERSION="0.10.3"
+curl --silent --remote-name https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_amd64.zip
```

Comment:
It might be nice to more explicitly call out somewhere that this walk-through presumes an OS with a Linux AMD64 architecture, and maybe to even give an example or two of such an OS so if they were selecting it in AWS they would be able to find it quickly and easily.

Reply:
Line 15 mentions this guide is for Linux hosts running systemd. Does that need to be more prominent or reinforced somewhere else? 

A quick check of instances on AWS and I can't find any non-amd64 Linux variants in the lists without specifically searching for them, so I'm inclined to think anyone not knowing which OS to use will pick Amazon Linux, RHEL, Suse or Ubuntu (from the quick start screen). These instructions have been tested on Ubuntu but I'm not aware of any reason they wouldn't work on the others as we don't use package managers in the deployment guide.

Reply:
Oh no! I do remember that line 15 says that. That's good. Thanks!

On the OS, it sounds like the AWS UI puts the architecture at the forefront. I can't remember offhand. I know my Linux, which is a variant of Ubuntu, _is not_ AMD64, so that's why I was asking. However, if there's no way to accidentally go off the rails here in the AWS UI, probably good to go.